install/kubernetes/helm/subcharts/mixer/values.yaml

```diff
@@ -7,7 +7,6 @@ env:
   GODEBUG: gctrace=2
 
 policy:
-  enabled: true
```

Comment:
is this needed in this PR?

Reply:
I'd say yes because user may not want to have policy installed, see https://preliminary.istio.io/docs/setup/kubernetes/minimal-install/

Reply:
it's easy to disable it in an override. my question is, how does this fix the sync issue?

Reply:
I was just trying to clean up sources of confusion (boy scout motto and all). These are duplicated flags and I want to eliminate any potential sources of error.

Reply:
Basically this makes it clear which `mixer.policy.enabled` param is being used everywhere (i think).

Reply:
A quick search shows that pilot and galley both have this `enabled: true` in their values. So this seems odd. Note, I have no qualification to say what's right in helm.

Reply:
I discussed this with @sdake . No one seems to know why we have all those values duplicated or what the impact is (was my interpretation, at least). 

@sdake OK to remove the subcharts duplicates?

Reply:
I believe it is okay to remove the sub charts duplicates.